baybe/campaign.py

```diff
@@ -28,6 +28,7 @@
     telemetry_record_value,
 )
 from baybe.utils.boolean import eq_dataframe
+from baybe.utils.plotting import create_str_representation
```

Comment:
(Putting this here such that we can discuss)

@Scienfitz @AdrianSosic everything fixed, please have a look and see if this is now better. If there are any further comments feel free to make suggestions. I tested this locally and it now looks fine.

Reply:
pic?

Reply:
![image](https://github.com/user-attachments/assets/53e12aa2-c83c-45c1-b96d-1624a55937f8)
![image](https://github.com/user-attachments/assets/805621ce-9053-40f8-bc4a-7cbd302534a4)

Reply:
`Obejctive` is on the same level as `Search Space`, my screen is just not large enough to capture everything

Reply:
looks good from my side

Reply:
@AdrianSosic how about you, specifically the approach I now implemented to avoid hardcoding stuff?

Reply:
You have addressed one of the points I shared via teams, but the others are still open. will close here and rather open separate issues at the corresponding places

Reply:
@AdrianSosic so where are those issues? I don't see any

Reply:
see teams message